data/event.py

```diff
@@ -1,305 +1,535 @@
 
     @property
-    def download_allowed(self) -> bool:
-        for tournament in self.tournaments.values():
-            if tournament.download_allowed:
-                return True
-        return False
+    def uniq_id(self) -> str:
+        return self.stored_event.uniq_id
 
     @property
-    def errors(self) -> list[str]:
-        return self.reader.errors
+    def name(self) -> str:
+        return self.stored_event.name if self.stored_event.name else self.uniq_id
 
     @property
-    def warnings(self) -> list[str]:
-        return self.reader.warnings
+    def start(self) -> float:
+        return self.stored_event.start
 
     @property
-    def infos(self) -> list[str]:
-        return self.reader.infos
-
-    def _build_root(self):
-        section_key: str = 'event'
-        try:
-            section = self.reader[section_key]
-        except KeyError:
-            self.reader.add_error('rubrique absente', section_key)
-            return
+    def stop(self) -> float:
+        return self.stored_event.stop
 
-        key = 'css'
-        try:
-            self.css = section[key]
-        except KeyError:
-            self.reader.add_debug('option absente', section_key, key)
-
-        key = 'name'
-        default_name = self.uniq_id
-        try:
-            self.name = section[key]
-            if not self.name:
-                self.reader.add_error('option vide', section_key, key)
-                return
-        except KeyError:
-            self.name = default_name
-            self.reader.add_info(
-                f'option absente, par défaut [{default_name}]',
-                section_key,
-                key
-            )
-        except TypeError:
-            # NOTE(Amaras) This could happen because of a TOC/TOU bug
-            # https://en.wikipedia.org/wiki/Time-of-check_to_time-of-use
-            # After this, the section has already been retrieved, so no future
-            # access will throw a TypeError.
-            self.reader.add_error(
-                'la rubrique est devenue une option, erreur fatale',
-                section_key
-            )
-            return
+    @property
+    def formatted_start_date_time(self) -> str:
+        return format_timestamp_date_time(self.start)
+
+    @property
+    def formatted_start_date(self) -> str:
+        return format_timestamp_date(self.start)
+
+    @property
+    def formatted_start_time(self) -> str:
+        return format_timestamp_time(self.start)
+
+    @property
+    def formatted_stop_date_time(self) -> str:
+        return format_timestamp_date_time(self.stop)
+
+    @property
+    def formatted_stop_date(self) -> str:
+        return format_timestamp_date(self.stop)
+
+    @property
+    def formatted_stop_time(self) -> str:
+        return format_timestamp_time(self.stop)
+
+    @property
+    def players_number(self) -> int:
+        return sum([len(tournament.players_by_name_with_unpaired) for tournament in self.tournaments_by_id.values()])
```

Comment:
Use generator expressions
```suggestion
        return sum((len(tournament.players_by_name_with_unpaired) for tournament in self.tournaments_by_id.values()))
```